src/senaite/core/browser/controlpanel/sampleconditions/view.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, context, request):
                     u"listing_sampleconditions_column_description",
                     default=u"Description"
                 ),
-                "index": "description",
+                "index": "Description",
```

Comment:
I've seen that the `Description` index is a `ZCTextIndex`, which can not be used for sorting purposes.Therefore, we should simply remove this one.

Reply:
Addressed with https://github.com/senaite/senaite.core/pull/2501/commits/8f965efb495f4984b0fc5e70765113fc7acb1371 (removed from all listings from controlpanel)